portfolio/src/main/webapp/index.html

```diff
@@ -46,20 +46,27 @@
             Service
           </a>
         </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#forum">
+            Forum
+          </a>
+        </li>
       </ul>
     </div>
   </nav>
 
 </head>
 
-<body id="homepage">
+<body id="homepage" onload="populateComments()">
   <div class="container">
     <div class="row py-5 mb-3" style="height:100%">
       <div class="col-sm-12 my-auto mx-auto" style="padding-right:0px">
         <div class="container">
           <div class="row h-100 justify-content-center pt-5">
             <div class="col-md-6 my-auto">
-              <h2 id="greeting"></h2>
+              <h2>
+                Howdy! I'm Dan,
+              </h2>
```

Comment:
Optional/FYI: not every tag needs to break line. ```<h2>Howdy! I'm Dan,</h2>``` is fine if the tag contents are short.